pkg/controller/smtpcredentialset/smtpcredentialset_controller.go

```diff
@@ -116,8 +116,8 @@ func (r *ReconcileSMTPCredentialSet) reconcile(ctx context.Context, request reco
 			if err = p.DeleteSMTPCredentials(ctx, instance); err != nil {
 				return reconcile.Result{}, errorUtil.Wrapf(err, "failed to run delete handler for smtp credentials instance %s", instance.Name)
 			}
-			r.logger.Infof("deletion handler for smtp credential instance %s successful, ending reconciliation", instance.Name)
-			return reconcile.Result{}, nil
+			r.logger.Infof("waiting on successful deletion handler for smtp credential instance %s", instance.Name)
```

Comment:
Can we change this log to be similar to the other controllers? `Waiting for SMTP credentials to successfully delete`